.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,71 @@
+name: E2E tests
+
+on:
+  schedule:
+    - cron: "0 12 * * 1"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-docker-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-docker-${{ github.sha }}
+            ${{ runner.os }}-buildx-docker-
+            ${{ runner.os }}-
+
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Prepare Tags
+        id: tags
+        run: |
+          TAG=$(echo $GITHUB_SHA | head -c7)
+          IMAGE="dev/test"
+          echo ::set-output name=tagged_image::${IMAGE}:${TAG}
+          echo ::set-output name=tag::${TAG}
+
+
+      - uses: docker/setup-buildx-action@v1
+        id: buildx
+
+      - name: Build
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          file: ./Dockerfile
+          builder: ${{ steps.buildx.outputs.name }}
+          push: false
+          load: true
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new
+          tags: ${{ steps.tags.outputs.tagged_image }}
+
+      - name: Run sandbox
+        run: docker run --rm --name my-sandbox --detach -p 20000:20000 tqtezos/flextesa:20210602 flobox start
+
+      - uses: addnab/docker-run-action@v3
+        with:
+          image: ${{ steps.tags.outputs.tagged_image }}
+          docker_network: host
+          run: |
+            tezos-client config reset
+            tezos-client --endpoint http://my-sandbox:20000 bootstrapped
+            tezos-client --endpoint http://my-sandbox:20000 config update
+            tezos-client import secret key alice unencrypted:edsk3QoqBuvdamxouPhin7swCvkQNgq4jP5KZPbwWNnwdZpSpJiEbq
+            tezos-client import secret key bob unencrypted:edsk3RFfvaFaxbHx8BMtEW1rKQcPtDML3LXjNqMNLCzC3wLC1bWbAt
+            cd dev
+            nohup python3 test-zeekoe.py merch-setup --url "http://my-sandbox:20000" -v &
+            nohup python3 test-zeekoe.py cust-setup --url "http://my-sandbox:20000" -v &
+            python3 test-zeekoe.py scenario --channel 1 -v --command-list establish pay pay pay_all close
+            python3 test-zeekoe.py scenario --channel 2 -v --command-list establish pay store pay restore close
```

Comment:
Yes, not sure how to split it. I agree we shouldn't start a sandbox in the Dockerfile, this would be clutter, in particular in the future when building on top of other chains.